spec/support/factory_bot.rb

```diff
@@ -1,7 +1,3 @@
-require "factory_bot"
-
-FactoryBot.find_definitions
```

Comment:
removed that because with that method call, I couldn't add new factories (they were registed twice and I got Factory already registed error).